pycrostates/segmentation/segmentation.py

```diff
@@ -147,6 +156,87 @@ def compute_parameters(
             return_dist=return_dist,
         )
 
+    def compute_transition_probabilities(
+        self, stat="probability", ignore_self=True
+    ):
+        """Compute transition probabilities of microstate transitions.
+
+        Parameters
+        ----------
+        stat : str
+            Aggregate statistic to compute each transition. Can be:
+
+            * ``count``: show the number of observations of each transition.
+            * ``probability`` or ``proportion``: normalize such probabilities
+                along the first axis is always equal to 1.
+            * ``percent``: normalize such probabilities along the first axis
+                is always equal to 100.
```

Comment:
```suggestion
            - ``count``: show the number of observations of each transition.
            - ``probability`` or ``proportion``: normalize such probabilities
                along the first axis is always equal to 1.
            - ``percent``: normalize such probabilities along the first axis
                is always equal to 100.
```

Still not good, let's try with a different bullet point style since stars are also used for bold/italic/...